src/main/java/org/jitsi/jicofo/jigasi/TranscriberManager.java

```diff
@@ -0,0 +1,292 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.jicofo.jigasi;
+
+import net.java.sip.communicator.impl.protocol.jabber.extensions.jitsimeet.*;
+import net.java.sip.communicator.impl.protocol.jabber.extensions.rayo.*;
+import net.java.sip.communicator.service.protocol.*;
+import net.java.sip.communicator.service.protocol.event.*;
+import org.jitsi.jicofo.*;
+import org.jitsi.protocol.xmpp.*;
+import org.jitsi.util.*;
+import org.jivesoftware.smack.packet.IQ;
+import org.jivesoftware.smack.packet.Presence;
+import org.jxmpp.jid.*;
+
+/**
+ * The {@link TranscriberManager} class is responsible for listening to
+ * {@link ChatRoomMemberPropertyChangeEvent}s to see whether a
+ * {@link ChatRoomMember} is requesting a transcriber by adding a
+ * {@link TranscriptionLanguageExtension} to their {@link Presence}.
+ *
+ * @author Nik Vaessen
+ */
+public class TranscriberManager
+    implements ChatRoomMemberPropertyChangeListener
+{
+    /**
+     * The logger of this class.
+     */
+    private final static Logger logger
+        = Logger.getLogger(TranscriberManager.class);
+
+    /**
+     * The {@link ChatRoom} of the conference this class is managing
+     */
+    private ChatRoom chatRoom;
+
+    /**
+     * The {@link JigasiDetector} responsible for determining which Jigasi
+     * to dial to when inviting the transcriber.
+     */
+    private JigasiDetector jigasiDetector;
+
+    /**
+     * The {@link XmppConnection} used to Dial Jigasi.
+     */
+    private XmppConnection connection;
+
+    /**
+     * The transcription status; either active or inactive based on this boolean
+     */
+    private volatile boolean active;
+
+    /**
+     * Create a {@link TranscriberManager} responsible for inviting Jigasi as
+     * a transcriber when this is desired.
+     *
+     * @param protocolProviderHandler the handler giving access a XmppConnection
+     * @param chatRoom the room of the conference being managed
+     * @param jigasiDetector detector for Jigasi instances which can be dialed
+     * to invite a transcriber
+     */
+    public TranscriberManager(ProtocolProviderHandler protocolProviderHandler,
+                              ChatRoom chatRoom,
+                              JigasiDetector jigasiDetector)
+    {
+        this.connection
+            = protocolProviderHandler.getOperationSet(
+            OperationSetDirectSmackXmpp.class).getXmppConnection();
+
+        this.chatRoom = chatRoom;
+        this.jigasiDetector = jigasiDetector;
+    }
+
+    /**
+     * Initialise the manager by starting to listen for
+     * {@link ChatRoomMemberPropertyChangeEvent}s.
+     */
+    public void init()
+    {
+        chatRoom.addMemberPropertyChangeListener(this);
+
+        logger.info("initialised transcriber manager");
```

Comment:
Can we change the level to debug? Also, do we really need a separate `init` method (instead of adding the listener inline in the constructor)?